apps/nextjs-website/src/components/molecules/ApiSection/ApiSection.tsx

```diff
@@ -1,7 +1,16 @@
 import { useMemo, useState } from 'react';
+import {
+  Link,
+  MenuItem,
+  Select,
+  SelectChangeEvent,
+  Stack,
+  Typography,
+  useTheme,
+} from '@mui/material';
 import { ApiViewer } from '@/components/atoms/ApiViewer';
 import { Product } from '@/lib/types/product';
-import { MenuItem, Select, SelectChangeEvent, Stack } from '@mui/material';
+import { IconName, getIconFromName } from '@/helpers/getIconFromName';
 import { styles } from '@/components/molecules/ApiSection/ApiSection.styles';
 
```

Comment:
```suggestion
import Link from 'next/link';
import { ButtonNaked } from '@pagopa/mui-italia';

```

Reply:
if you accept the <ButtonNaked> suggestion, then you should accept this one as well.